mobile/src/Student/screens/StudentName/index.js

```diff
@@ -86,7 +97,8 @@ const StudentName = ({ navigation, gameSession, setTeamInfo }) => {
                                         placeholder={"First Name"}
                                         placeholderTextColor={colors.primary}
                                         ref={(ref) => {
-                                            this.firstNameTextRef = ref
+                                            //this.firstNameTextRef = ref
+                                            firstNameInput = ref
```

Comment:
Both here and below, I believe it should still be `firstNameTextRef` as that gives us a references to the text field and then we need to focus on it. Also, where `firstNameInput` is defined? Same is true for the password

Reply:
When I was looking into it, it said that 'firstNameTextRef' is declared but its value was never used so I just commented it out. firstNameInput (and lastNameInput) is declared at the top of the file on lines 11 and 12:
"  const [firstName, setFirstName] = useState("")
    const [lastName, setLastName] = useState("") "
I essentially replicated the format that was used for taking in user inputs in the enter game code screen (which used state instead of ref for user input):
"
const [gameCode, setGameCode] = useState("")
const [showErrorText, setShowErrorText] = useState(false)
"